### PR TITLE
Add base dockerfile with FSL and dicom2niix

### PR DESCRIPTION
**Pull request by yu-pri**

### CODE REVIEW

File: docker/neurodocker_gen_base.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+docker run --rm repronim/neurodocker:latest \
+  generate docker \
+    --base-image ubuntu:22.04 \
+    --pkg-manager apt \
+    --dcm2niix version="v1.0.20201102" \
+    --fsl version=6.0.6.4 \
```
Comment (yu-pri) [inline docker/neurodocker_gen_base.sh:8]: versions

File: docker/neurodocker_gen_base.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    --yes \
+    --miniconda version=latest \
+                conda_install="python=3.10 traits" \
+                pip_install="nipype" > ./Dockerfile.fsl
```
Comment (yu-pri) [inline docker/neurodocker_gen_base.sh:12]: python 3.10 and miniconda

File: docker/neurodocker_gen_base.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+docker run --rm repronim/neurodocker:latest \
```
Comment (yu-pri) [inline docker/neurodocker_gen_base.sh:3]: The dockerfile is completely autogenerated, minimal maintenance.
Couldn't get SPM to work in a generated image, may need manual maintenance. But then it's not part of our workflow yet. Get back to it when we actually need it.

File: docker/Dockerfile.fsl [hunk 4]
```diff
@@ -0,0 +1,92 @@
+# Generated by Neurodocker and Reproenv.
```
Comment (yu-pri) [inline docker/Dockerfile.fsl:1]: No manual tweaks made.

File: docker/neurodocker_gen_base.sh [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  generate docker \
+    --base-image ubuntu:22.04 \
```
Comment (yu-pri) [inline docker/neurodocker_gen_base.sh:5]: Not happy with using a ubuntu image, kinda bloated. But it's the only one that works out-of-the-box.

File: docker/README.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+# fMRI in docker
+
+## Usage
+
+```shell
+docker run -it yupri/fmri:fsl-latest
+```
```
Comment (yu-pri) [inline docker/README.md:7]: Didn't upload to Docker Hub yet. If you want to play around -- follow step 2 and further below.

**Review by IvryaB — APPROVED** [review]
Everything good)

**Pull request merged**